src/hotspot/os/posix/perfMemory_posix.cpp

```diff
@@ -515,8 +524,9 @@ static char* get_user_name(uid_t uid) {
 //
 // the caller is expected to free the allocated memory.
 //
-// If nspid != -1, look in /proc/{vmid}/root/tmp for directories
-// containing nspid, otherwise just look for vmid in /tmp
+// On Linux, if nspid != -1, look in /proc/{vmid}/root/tmp for directories
```

Comment:
Please move this comment block into the ifdef where the code is.

Reply:
done